requirements/edx/local.txt

```diff
@@ -1,12 +1,13 @@
 # Python libraries to install that are local to the edx-platform repo
 -e .
--e common/lib/calc
+
 -e common/lib/capa
--e common/lib/chem
--e common/lib/dogstats
--e common/lib/safe_lxml
--e common/lib/sandbox-packages
--e common/lib/symmath
 -e common/lib/xmodule
 
+-e openedx/core/lib/calc
```

Comment:
I believe that this won't work as the code currently pip installs these libraries separately, so they get their own namespace. It is possible that this was only necessary when this code lived under common, and under openedx we could switch these over to no longer be pip installed. If we did that, then we could use imports as you suggest.

@robrap @nasthagiri what do you think?